res/translations/strings.de.po

```diff
@@ -1746,7 +1747,7 @@ msgstr "DEBUG"
 
 msgctxt "spell"
 msgid "goodwinds"
-msgstr "Beschwörung eines Wasserelementares"
+msgstr "Beschwöre einen Wasserelementar"
```

Comment:
Das habe ich schon angefasst, und bin am Ende bei "Wasserlementar" etc. gelandet. Commit ist bereits in devlop, mit Code, der die alten Namen als Alternativen versteht.